src/domain/common.ts

```diff
@@ -1,5 +1,17 @@
+import { ARBITRUM, AVALANCHE } from "config/chains";
+
 export function get1InchSwapUrl(chainId: number, from?: string, to?: string) {
   const rootUrl = `https://app.1inch.io/#/${chainId}/unified/swap`;
   if (!from && !to) return rootUrl;
   return `${rootUrl}/${from}/${to}`;
 }
+
+export function getLeaderboardLink(chainId) {
```

Comment:
I'm afraid that "common" will turn into a new Helpers.js, because it's a very general name. Now some kind of link building utilities are being put together here, maybe we should call it (domain/routes/utils.ts)? Maybe even put it in config/routes.ts - because there's no special logic / types / hooks but just using parameters with hardcoded urls

Reply:
Yeah makes sense to put in the config. Will update